packages/xo-web/src/common/xo/index.js

```diff
@@ -2269,14 +2269,21 @@ export const deleteSr = sr =>
 export const fetchSrStats = (sr, granularity) => _call('sr.stats', { id: resolveId(sr), granularity })
 
 export const forgetSr = sr =>
-  confirm({
-    title: _('srForgetModalTitle'),
-    body: _('srForgetModalMessage'),
-  }).then(() => _call('sr.forget', { id: resolveId(sr) }), noop)
+  forgetSrs([sr])
+
 export const forgetSrs = srs =>
   confirm({
     title: _('srsForgetModalTitle'),
```

Comment:
Since we're calling this function from `forgetSr`, maybe we should adjust the title of the modal when there's only one SR, so that it doesn't say "Forget selected SRs" when we're forgetting the SR from the SR view.